### PR TITLE
Cleaned up patch for #55.

### PR DESCRIPTION
**Pull request by fawick**
Changes to _test.go were not part of the patch. I added them to be able
to run the tests after patching. 

All other changes are from #55, https://github.com/restic/restic/files/133561/snapshot-comments-v1.diff.txt by @mappu

### CODE REVIEW

File: src/restic/cmd/restic/cmd_snapshots.go [hunk 1]
```diff
@@ -119,13 +119,7 @@ func (cmd CmdSnapshots) Execute(args []string) error {
 			continue
 		}
 		id := sn.ID()
-		tab.Rows = append(tab.Rows, []interface{}{hex.EncodeToString(id[:plen/2]), sn.Time.Format(TimeFormat), sn.Hostname, sn.Paths[0]})
-
-		if len(sn.Paths) > 1 {
```
Comment (fd0) [inline src/restic/cmd/restic/cmd_snapshots.go]: This is strange. @mappu why did you remove the other paths from the table?

**Comment by fd0** [comment]
Looks good, thanks for your work @fawick!

We'll need to refactor this sometime later, we can't just keep adding parameters to the `Snapshot()` function...

File: src/restic/cmd/restic/cmd_snapshots.go (hunk 1 above)
Comment (mappu) [inline src/restic/cmd/restic/cmd_snapshots.go]: With 80 character default terminal width it was unreadable to have all fields visible.

**Comment by fd0** [comment]
I've thought about this patch, and I've decided not to merge it in this form. Adding the comment field is nice, but removing the most important thing from the snapshot output, the directories that have been backed up, is not good.

Thanks for your work!
